docs/tutorials/11_VarQTE.ipynb

```diff
@@ -0,0 +1,957 @@
+    "$$\n",
+    "\n",
+    "where\n",
+    "\n",
+    "$$F_{ij}=\\text{Re}\\left[\\frac{\\partial\\left\\langle \\psi\\left[\\theta\\right]\\right|}{\\partial\\theta_{i}}\\frac{\\partial\\left|\\psi\\left[\\theta\\right]\\right\\rangle }{\\partial\\theta_{j}}+\\frac{\\partial\\left\\langle \\psi\\left[\\theta\\right]\\right|}{\\partial\\theta_{i}}\\left|\\psi\\left[\\theta\\right]\\right\\rangle \\left\\langle \\psi\\left[\\theta\\right]\\right|\\frac{\\partial\\left|\\psi\\left[\\theta\\right]\\right\\rangle }{\\partial\\theta_{j}}\\right]$$,\n",
+    "\n",
+    "and\n",
+    "\n",
+    "$$V_{i}=\\begin{cases}\n",
+    "\\text{Im}\\left[\\frac{\\partial\\left\\langle \\psi\\left[\\theta\\right]\\right|}{\\partial\\theta_{i}}H\\left|\\psi\\left[\\theta\\right]\\right\\rangle +\\left\\langle \\psi\\left[\\theta\\right]\\right|\\frac{\\partial\\left|\\psi\\left[\\theta\\right]\\right\\rangle }{\\partial\\theta_{i}}\\left\\langle H\\right\\rangle _{\\theta}\\right] & \\text{Real time}\\\\\n",
+    "-\\text{Re}\\left[\\frac{\\partial\\left\\langle \\psi\\left[\\theta\\right]\\right|}{\\partial\\theta_{i}}H\\left|\\psi\\left[\\theta\\right]\\right\\rangle \\right] & \\text{Imaginary time}\n",
+    "\\end{cases}$$."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "caoBgyluhQ3L"
+   },
+   "source": [
+    "## Running VarQTE"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "WG-18slyLnVM"
+   },
+   "source": [
+    "In this tutorial, we will use two classes that extend `VarQTE`, `VarQITE` ([Variational Quantum Imaginary Time Evolution](https://qiskit.org/ecosystem/algorithms/stubs/qiskit_algorithms.VarQITE.html#qiskit_algorithms.VarQITE)) and `VarQRTE` ([Variational Quantum Real Time Evolution](https://qiskit.org/ecosystem/algorithms/stubs/qiskit_algorithms.VarQRTE.html)) for time evolution.\n",
+    "We can use a simple Ising model on a spin chain to illustrate this. Let us consider the following Hamiltonian:\n",
+    "\n",
+    "$$H = -J\\sum_{i=0}^{L-2}Z_{i}Z_{i+1} - h\\sum_{i=0}^{L-1}X_i$$\n",
+    "\n",
+    "where $J$ stands for the interaction energy, and\n",
+    "$h$ represents an external field which is orthogonal to the transverse direction. $Z_i$ and $X_i$ are the Pauli operators on the spins. Taking $L=2$, $J=0.2$,  and $h =1$, the Hamiltonian and the magnetization $\\sum_i Z_i$ can be constructed using `SparsePauliOp` as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "id": "6B0nZEDh0EqI"
+   },
+   "outputs": [],
+   "source": [
+    "from qiskit.quantum_info import SparsePauliOp\n",
+    "\n",
+    "hamiltonian = SparsePauliOp(['ZZ',  'IX', 'XI'],\n",
+    "              coeffs=[-0.2 , -1, -1])\n",
+    "\n",
+    "magnetization = SparsePauliOp([ 'IZ', 'ZI'], coeffs=[1, 1])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Y_9Jskw4m8nF"
+   },
+   "source": [
+    "# Imaginary Time Evolution\n",
```

Comment:
Can we drop this heading level and the later one for Real Time Evolution below to a 2nd level i.e ## There should only be one at the top level which is the main title. I did not do a suggestion as I think other headings will need to be adjusted down too. Otherwise, as it stands, with going through the tutorials in the html down the sidebar I figured I was missing something as it seems we have more than we actually have - as you can see in this image

![image](https://github.com/qiskit-community/qiskit-algorithms/assets/40241007/85d02826-d356-4901-8ef4-22e4948d27d5)